README.md

```diff
@@ -71,3 +71,37 @@ Update the version number in the local package.json file so that it interacts co
 
 ### Inputs
 - `version` - The new active version number of the package
+
+## [build-with-ubp](https://github.com/mindjolt/uc-actions/tree/main/build-with-ubp)
+
+Create a build using UBP, supports both UnityCore version and UC-UBP-SDK versions.
+
+### Inputs
+- `files` - The files you want to build with space delimited
```

Comment:
This could probably be wordsmithed better. Are they JSON files? Perhaps we could rename the input to include that if we are expecting them to be JSONs.

`A space delimited list of JSON configuration files`